golang/lib/execution/lambda_executor.go

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright (c) 2020 - present Kurtosis Technologies LLC.
+ * All Rights Reserved.
+ */
+
+package execution
+
+import (
+	"github.com/kurtosis-tech/kurtosis-client/golang/kurtosis_core_rpc_api_bindings"
+	"github.com/kurtosis-tech/kurtosis-client/golang/lib/networks"
+	"github.com/kurtosis-tech/kurtosis-client/golang/lib/services"
+	"github.com/kurtosis-tech/kurtosis-lambda-api-lib/golang/kurtosis_lambda_docker_api"
+	"github.com/kurtosis-tech/kurtosis-lambda-api-lib/golang/kurtosis_lambda_rpc_api_bindings"
+	"github.com/kurtosis-tech/kurtosis-lambda-api-lib/golang/kurtosis_lambda_rpc_api_consts"
+	"github.com/kurtosis-tech/minimal-grpc-server/server"
+	"github.com/palantir/stacktrace"
+	"google.golang.org/grpc"
+	"time"
+)
+
+const (
+	grpcServerStopGracePeriod = 5 * time.Second
+)
+
+type LambdaExecutor struct {
+	apiContainerSocket        string
```

Comment:
we're not going to want to take in apiContaienrSocket or serializedCustomParamsStr as constructor params anymore; there's an even cooler way to do this. I'll do it for the testsuite first, and then you can use it here

Reply:
This: https://github.com/kurtosis-tech/kurtosis-testsuite-api-lib/pull/2

Reply:
I like it, it's really a good improvement.